scripts/validate.py

```diff
@@ -58,6 +59,13 @@ def load_schema(name):
     return ALL_SCHEMAS[name]
 
 
+def load_schema_exceptions(message_template):
+    if message_template in ["update_action"]:
```

Comment:
Why not `if message_template = "update_action"`? Would there be any additions in the future?

Reply:
I guess not.